Dockerfile

```diff
@@ -163,9 +163,8 @@ RUN --mount=type=cache,sharing=shared,id=cargo_registry_index,target=${CARGO_HOM
     # Meanwhile if you want to update wasm-bindgen you also need to update version in:
     #  - packages/wasm-dpp/Cargo.toml
     #  - packages/wasm-dpp/scripts/build-wasm.sh
-    cargo install --profile "$CARGO_BUILD_PROFILE" wasm-bindgen-cli@0.2.86
+    cargo install --profile "$CARGO_BUILD_PROFILE" wasm-bindgen-cli@0.2.92 trunk
```

Comment:
What tunk for?

Reply:
Ok, it's to build a visualizer. It's going to be as annoying as it was with random documents. We eventually moved random documents to a separate repo. Please put the visualizer under the feature and disable it by default. We can enable this feature in the docker container optionally.